### ISSUE TITLE
Links should have sufficient cues to suggest clickability

### ISSUE THREAD

**Issue by sdellis** [projectblacklight/arclight#488]
There are a number of links (take [this page](https://arclight-demo.projectblacklight.org/catalog/reg_003-xmlref1885) for example) that are not apparent until you hover over them since they are the same color and weight as other non-linked text.

This NNG article contains supporting evidence for this request:
https://www.nngroup.com/articles/clickable-elements/
